### DISCUSSION TITLE
Can files be automatically accepted?

### DISCUSSION THREAD

**Discussion by WarlockFish** [schlagmichdoch/PairDrop#223] [General]
Can I automatically accept files after sending file？

**Comment by schlagmichdoch** [comment]
If you pair your device and open the Edit Paired Devices dialog, there is an option to auto-accept transfer requests from that device.

**Reply by WarlockFish** [reply]
auto-accept transfer is ok, but I find  the first time is fine, the second time i need to click close before transferring files 
![图片](https://github.com/schlagmichdoch/PairDrop/assets/30708115/475c5088-396e-4da6-b3e6-cc91e4be6fe9)

**Reply by schlagmichdoch** [reply]
What do you mean with „the second time“?
Could you try to be more precise by using „sender“ and „receiver“ or including a short screen record? Otherwise it’s kind of hard to understand what you mean

**Reply by WarlockFish** [reply]
[录屏 2023年12月15日 00时58分05秒.webm](https://github.com/schlagmichdoch/PairDrop/assets/30708115/398235eb-9984-4a02-a4f8-2bb0a92ab6c6)

**Comment by schlagmichdoch** [comment]
I have forgotten this, sorry! I think this is linked to #40, let's move the discussion over there. Feel free to test whether the current testing instance that I posted on the issue resolves your problem.
